libtsuba/src/RDG.cpp

```diff
@@ -613,29 +590,16 @@ tsuba::RDG::Store(
         rdg_dir(), handle.impl_->rdg_manifest().dir()));
   }
 
-  auto desc_res = WriteGroup::Make();
-  if (!desc_res) {
-    return desc_res.error();
-  }
   // All write buffers must outlive desc
-  std::unique_ptr<WriteGroup> desc = std::move(desc_res.value());
+  auto desc = KATANA_CHECKED(WriteGroup::Make());
```

Comment:
nit:
I'm sure this is up for debate, but I prefer the explicit `std::unique_ptr<WriteGroup>` here vs the `auto`. I don't like having to look at the signature of `WriteGroup::Make()` of whatever we are calling to know what `desc` is. Especially since its not just a `WriteGroup` like one might assume, but a  `std::unique_ptr<WriteGroup>`.

Reply:
@witchel wanted to get your opinion on this. I'm a fan of using `auto` for most things since it's usually cleaner to write, but I can also see Evan's point here.

Reply:
This is an ongoing debate, but I think the non-auto camp is winning at Katana.  I see the value of explicit types for people less familiar with the code.  We should prioritize the needs of this audience.

Reply:
Makes sense. I will make the changes accordingly